apps/frontend-v3/lib/components/navs/BuildNavLink.tsx

```diff
@@ -43,6 +44,7 @@ export function BuildNavLink() {
   }, [isOpen, onClose])
 
   const handleToggle = () => {
+    trackEvent(AnalyticsEvent.ClickNavBuild)
```

Comment:
The track event code does not seem to differentiate between prod and the dev environments. That could probably skew the results a little.

Reply:
I see there is some code before that which mades it only run on prod:

```    
// Optional: Only track on production; remove these two lines if you want to track other environments
if (!isProd) return
```

The `zen.balancer.fi` does not point to anything though, we can remove it.